client/reader/tag-stream/controller.js

```diff
@@ -33,25 +38,33 @@ export const tagListing = ( context, next ) => {
 		context.headerSection = renderHeaderSection();
 	}
 	context.primary = (
-		<AsyncLoad
-			require="calypso/reader/tag-stream/main"
-			key={ 'tag-' + encodedTag }
-			streamKey={ streamKey }
-			encodedTagSlug={ encodedTag }
-			decodedTagSlug={ tagSlug }
-			trackScrollPage={ trackScrollPage.bind(
-				// eslint-disable-line
-				null,
-				basePath,
-				fullAnalyticsPageTitle,
-				analyticsPageTitle,
-				mcKey
-			) }
-			startDate={ startDate }
-			onUpdatesShown={ trackUpdatesLoaded.bind( null, mcKey ) } // eslint-disable-line
-			showBack={ !! context.lastRoute }
-			showPrimaryFollowButtonOnCards={ false }
-		/>
+		<>
+			<DocumentHead
+				title={ translate( 'Articles About %s – Reader – WordPress.com', {
```

Comment:
For keeping this more consistent with the rest of Reader:

```suggestion
				title={ translate( 'Articles About %s ‹ Reader', {
```